### ISSUE TITLE
Main Account (Etherbase) keys? Help!

### ISSUE THREAD

**Issue by willdady** [ethereum/mist#2541]
Where are the private keys stored for the Main Account (Etherbase) account? When I go to _Accounts > Backup > Accounts_ the one UTC file I have in the keystore directory has a different public key than the one shown for the Main Account in Mist. I was working under the assumption that this is the private key file for this 'Main Account'. I'm obviously concerned as I've sent some funds to this main account but can't figure out where the private key is for it.

Sure enough when I load the UTC file using myetherwallet.com and decrypt it using the password I created in Mist I get an account with the public key matching the UTC file but not the same account as 'Main Account'. Any help would be much appreciated.

**Comment by willdady** [comment]
OK so thankfully I've found the key.

I have Mist installed on **S:/** drive however, when I use the backup option in Mist it opens the following path, on **C:/** drive, C:\Users\will\AppData\Roaming\Ethereum. What's strange is there is **ALSO** a directory on **S:/** drive, S:\Users\will\AppData\Roaming\Ethereum. It's this one where I found the **actual** key for the Main Account after some thorough searching. 

I'm not 100% sure where the key found on **C:/** drive came from, it's possible I had Mist installed on that drive in the past but honestly can't remember.

What's confusing (albeit extremely concerning) is that Mist is installed on **S:/** drive and has created the key on **S:/** drive but the backup option opens the wrong directory on **C:/** drive. This seems very dangerous as I had incorrectly backed up the wrong key file and only figured out it was wrong by chance. This seems like a bug to me.

**Comment by jordanmack** [comment]
Why do you have a Users folder on C: and S:?

**Comment by willdady** [comment]
Like most people **C:/** is my OS drive so that's where the Users directory is by default. Mist has created this path **S:\Users\will\AppData\Roaming\Ethereum**. I know Mist has created this as there are no other files in any of those sub directories. It's like Mist assumes it's installed on the OS drive without actually checking which is why the path is created on the wrong drive. As mentioned above, the danger is the _Accounts > Backup > Accounts_ option opens the path on **C:/** drive, not the path to where the key is actually stored (at least in my case).

**Comment by jordanmack** [comment]
Could you open a command prompt and tell me the results of `echo %APPDATA%`?

**Comment by willdady** [comment]
`C:\Users\will\AppData\Roaming`

**Comment by jordanmack** [comment]
Have you had an older version of Mist installed on this computer before?

I setup a VM with a C drive and S drive to try to replicate this. When the installer asked where to put the files I specified S drive for both the program and data paths. It appears to have installed correctly to the paths specified. However, when I went into Mist and tried to open the menu `Accounts > Backup > Accounts`, nothing happens.

**Comment by willdady** [comment]
@jordanmack I can't be 100% sure I didn't have Mist installed on **C:\** at some point. That could explain why I had a key located there.

**Comment by evertonfraga** [comment]
Hi @willdady, I'm glad you managed to solve your problem. 

Furthermore, did you choose S:\ drive when installing mist from the Mist Installer? 

I'll close this for now, if you need anything else, just reopen this issue. Thanks!

**Comment by squibbler00** [comment]
I am having the same problem, but on a Mac.  Had ether sent to etherbase account, and I can't find the keystore folder.

**Comment by andrasfuchs** [comment]
I have exactly the same situation on Windows. I chose drive O as the data directory during the installation. I installed version 0.8.10 and I have never used Ethereum or Mist before.

My keystore and geth folders are on drive O, all the others on drive C's %APPDATA%. When I click on the Accounts / Backup / Accounts, it opens up the folder on **drive C**. 
I think it is a bug. I was lucky enough to realize the my keys are not there, but others may loose their ether if they backup the wrong folder.
